src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -41,6 +40,9 @@
 
 register_cli_argument('resource list', 'name', resource_name_type)
 register_cli_argument('resource move', 'ids', nargs='+')
+register_cli_argument('resource invoke-action', 'action', help='The action that will be invoked')
+register_cli_argument('resource invoke-action', 'parameters', help='Parameter arguments for the action that will be passed along in the post request body')
+register_cli_argument('resource invoke-action', 'post_url', help='The url that will be sent in the POST request (must include api-version query parameter)')
```

Comment:
Per on going mail thread, we might expose a general request url which you can apply any http method

Reply:
Sorry for changing the mind, but let us get rid of it.
We can move such functionalities into ms-rest-azure moving forward